bitwarden_license/bit-web/src/app/billing/providers/clients/vnext-manage-clients.component.ts

```diff
@@ -0,0 +1,201 @@
+import {
+  ManageClientNameDialogResultType,
+  openManageClientNameDialog,
+} from "./manage-client-name-dialog.component";
+import {
+  ManageClientSubscriptionDialogResultType,
+  openManageClientSubscriptionDialog,
+} from "./manage-client-subscription-dialog.component";
+import { vNextNoClientsComponent } from "./vnext-no-clients.component";
+
+@Component({
+  templateUrl: "vnext-manage-clients.component.html",
+  standalone: true,
+  imports: [
+    AvatarModule,
+    TableModule,
+    HeaderModule,
+    SharedOrganizationModule,
+    vNextNoClientsComponent,
+  ],
+})
+export class vNextManageClientsComponent implements OnInit {
+  providerId: string;
+  provider: Provider;
+  loading = true;
+  isProviderAdmin = false;
+  clients: ProviderOrganizationOrganizationDetailsResponse[];
+  dataSource: TableDataSource<ProviderOrganizationOrganizationDetailsResponse> =
+    new TableDataSource();
+
+  protected destroy$ = new Subject<void>();
+  protected searchControl = new FormControl("", { nonNullable: true });
+  protected plans: PlanResponse[];
+
+  constructor(
+    private billingApiService: BillingApiServiceAbstraction,
+    private providerService: ProviderService,
+    private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private dialogService: DialogService,
+    private i18nService: I18nService,
+    private toastService: ToastService,
+    private validationService: ValidationService,
+    private webProviderService: WebProviderService,
+  ) {}
+
+  ngOnInit(): void {
+    this.activatedRoute.parent.params
+      .pipe(
+        switchMap((params) => {
+          this.providerId = params.providerId;
+          return this.providerService.get$(this.providerId).pipe(
+            map((provider: Provider) => provider?.providerStatus === ProviderStatusType.Billable),
+            map((isBillable) => {
+              if (!isBillable) {
+                return from(
+                  this.router.navigate(["../clients"], {
+                    relativeTo: this.activatedRoute,
+                  }),
+                );
+              } else {
+                return from(this.load());
+              }
+            }),
+          );
+        }),
+        takeUntil(this.destroy$),
+      )
+      .subscribe();
+
+    this.searchControl.valueChanges
+      .pipe(debounceTime(200), takeUntil(this.destroy$))
+      .subscribe((searchText) => {
+        this.dataSource.filter = (data) =>
+          data.organizationName.toLowerCase().startsWith(searchText.toLowerCase());
+      });
+  }
+
+  async load() {
+    this.provider = await firstValueFrom(this.providerService.get$(this.providerId));
+
+    this.isProviderAdmin = this.provider.type === ProviderUserType.ProviderAdmin;
+
+    this.clients = (
+      await this.billingApiService.getProviderClientOrganizations(this.providerId)
+    ).data;
+
+    this.clients.map((client) => (client.plan = client.plan.replace(" (Monthly)", "")));
```

Comment:
Good call removing this from the template.

`map` returns a new array, it's clearer to use `forEach` when you're mutating the existing array:

```suggestion
    this.clients.forEach((client) => (client.plan = client.plan.replace(" (Monthly)", "")));
```